requirements.txt

```diff
@@ -1,8 +1,8 @@
-pydantic == 2.7.4
-bittensor == 7.4.0
-polygon-api-client == 1.13.7
+pydantic == 2.10.3
+git+https://github.com/jbonilla-tao/client-python.git@patch-1#egg=polygon-api-client
```

Comment:
what does this do?

Reply:
Until polygon approves this https://github.com/polygon-io/client-python/pull/802,

we will have to pull the source code from this fork which removes a package conflict with BT version >= 8.4.5